ovos_core/intent_services/padatious_service.py

```diff
@@ -149,15 +167,17 @@ def __init__(self, bus, config):
 
         if self.is_regex_only:
             if not _pd:
-                LOG.error('Padatious not installed. Falling back to pure regex alternative')
+                LOG.info('Padatious not installed. '
+                         'Falling back to pure regex alternative')
```

Comment:
update this log to mention padacioso explicitly, its far more than a simple regex now